bin/sl-run.txt

```diff
@@ -22,6 +22,9 @@ Options:
                      Disable timestamping of worker log lines by supervisor.
   --no-timestamp-supervisor
                      Disable timestamping of supervisor log messages.
+  --no-log-decoration
```

Comment:
I thought this would disable _all_ decoration (including timestamps), @rmg, what do you think? But I'm OK with it as-is, I think. It means special handling for timestamps, and everything else is lumped in as "decoration", which works for me.